code/game/turfs/turf.dm

```diff
@@ -111,11 +111,10 @@ GLOBAL_LIST_EMPTY(station_turfs)
 	var/turf/T = SSmapping.get_turf_above(src)
 	if(T)
 		T.multiz_turf_new(src, DOWN)
-		SEND_SIGNAL(T, COMSIG_TURF_MULTIZ_NEW, src, DOWN)
 	T = SSmapping.get_turf_below(src)
 	if(T)
 		T.multiz_turf_new(src, UP)
-		SEND_SIGNAL(T, COMSIG_TURF_MULTIZ_NEW, src, UP)
+
```

Comment:
Please add this part of code to `/turf/open/space/Initialize()` since it override `/turf/open/Initialize()`
[Here](https://github.com/tgstation/tgstation/blob/0ffa1473d99bd3ad9ec71fece24b3aa531fb063f/code/game/turfs/open/space/space.dm#L33-L68)